woof-code/3builddistro

```diff
@@ -493,6 +493,46 @@ for i in $(ls sandbox3/rootfs-complete/usr/share/locale); do
 	case $i in en|en_US|locale.alias) continue ;; esac
 	rm -rf sandbox3/rootfs-complete/usr/share/locale/$i
 done
+
+#overwrite current documentation if DOCS_URL present
+
+case $CACHE_PERSONAL_DOCS in #_00build.conf
+	y|n) echo -n ;;
+	*) CACHE_PERSONAL_DOCS=y ;;
+esac
+
+if [ -n "$DOCS_URL" ]; then
+	echo "Overwriting current documentation with developer provided $DOCS_URL..."
+	rm -rf sandbox3/rootfs-complete/usr/share/doc
+	
+	if [ "$CACHE_PERSONAL_DOCS" = 'y' ]; then
+		mkdir -p "../../local-repositories/personal_docs/$DOCS_URL"
```

Comment:
If `DOCS_URL` is `https://google.com/a/b/c`, does this line try to create `../../local-repositories/personal_docs/https://google.com/a/b/c"`? This won't work.

Reply:
It creates `https:/google.com/a/b/c`. I have tested this part of code in a seperate test directory. First `https:` directory is created, inside it `google.com` and so on.

I did it this way because I couldn't think of any better option.

Reply:
> I did it this way because I couldn't think of any better option.

Why not `${DOCS_URL##*/}`, as I suggested in some other comment?